RetailCoder.VBE/Inspections/QuickFixes/QuickFixHelper.cs

```diff
@@ -10,28 +10,32 @@
 
 namespace Rubberduck.Inspections.QuickFixes
 {
-    class AssignedByValParameterQuickFixHelper
+    class QuickFixHelper
     {
         private Declaration _target;
         private QualifiedSelection _selection;
-        private HashSet<string> _namesInUse;
+        private HashSet<string> _identifierNamesDeclaredInProcedureScope;
         private ICodeModule _module;
-        public AssignedByValParameterQuickFixHelper(Declaration target, QualifiedSelection selection)
+        public QuickFixHelper(Declaration target, QualifiedSelection selection)
         {
             _target = target;
             _selection = selection;
-            _namesInUse = new HashSet<string>();
+            _identifierNamesDeclaredInProcedureScope = new HashSet<string>();
             _module = selection.QualifiedName.Component.CodeModule;
         }
-        public string GetModuleLine(int line)
+        /// <summary>
+        /// Returns the CodeModule string located at lineNumber
+        /// of the newly inserted line.
+        /// </summary>
+        public string GetModuleLine(int lineNumber)
```

Comment:
That all sounds good to me.  I'll make those types of changes on the next pass.  Thanks.